.changeset/gentle-rockets-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+'renterd': patch
+---
+
+The file breadcrumb nav now shows the root as "Buckets".
```

Comment:
Any chance future PRs can be limited to 1 logical change per PR? Figuring out what is what is a bit tricky for people unfamiliar with the codebase and frontend development ^^

Reply:
Definitely